O365/account.py

```diff
@@ -110,7 +109,7 @@ def authenticate(self, *, scopes=None, **kwargs):
                 return False
 
         elif self.con.auth_flow_type == 'credentials':
-            return self.con.request_token(None, requested_scopes=scopes)
+            return self.con.request_token(None)
```

Comment:
This is wrong. The user can pass the scopes to the authenticate method and so those must be passed to the request_token method.

Reply:
In the case where auth_flow_type='authentication' the authenticate function permits only one assignment on the scopes. Either during the initialization of the Account or during the authenticate. That is why i thought that since for auth_flow_type='credentials', we assign the '.default' scope during Account initialization, it would not be valid to reassign the scope in the authentication.

Reply:
ok, what I can do is change the implementation to require the scopes to be passed into the account init in case auth_flow_type is credentials (as this is needed for the auto-refresh to work).

Then we can do what you did.

Reply:
If the '.default' scope is not needed then a similar approach to the auth_flow_type='authentication' could be implemented and require from the user to set the scopes in either init or authenticate.

Reply:
The .default is needed, but the user can also pass another scopes such as 'offline_access'.